checkmate/models/domain/__init__.py

```diff
@@ -0,0 +1 @@
+"""Logical domain model objects."""
```

Comment:
I think we can just put them all in `checkmate.models` directly like we did for lms. Especially since there aren't currently any non-persisted ones

Reply:
See comment above

Reply:
As mentioned elsewhere, I think this is worth talking about. I'd prefer to go the other way and split up LMS. I don't think the decision to mix the two was what we would do starting from scratch, but influenced by the existing code.